test/IRGen/ptrauth-dynamic_replaceable.sil

```diff
@@ -45,7 +45,7 @@
 // CHECK-NEXT:   ret void
 // CHECK-NEXT: }
 
-sil [dynamically_replacable] @test_dynamically_replaceable : $@convention(thin) () -> () {
+sil [dynamically_replaceable] @test_dynamically_replaceable : $@convention(thin) () -> () {
```

Comment:
Renaming this attribute is a cross-component change, so you may want to leave this out for now.

Reply:
Reverted